files/ifup-vxlan

```diff
@@ -13,11 +13,9 @@ need_config ${CONFIG}
 
 source_config
 
-/etc/sysconfig/network-scripts/ifup ${PHYS_DEV}
-
 if ! ip link show dev ${DEVICE} >/dev/null 2>&1 ; then
     echo "Creating vxlan ${DEVICE} - VNI: ${VNI}, IP Address: ${IPADDR}/${PREFIX}"
-    ip link add ${DEVICE} type vxlan id ${VNI} group ${GROUP} ttl ${TTL:-10} dev ${PHYS_DEV} dstport ${PORT:-4789}
+    ip link add ${DEVICE} type vxlan id ${VNI} group ${GROUP} ttl ${TTL:-10} dev ${PHYS_DEV} dstport ${DSTPORT:-4790}
```

Comment:
Do you need the default if `DSTPORT` is never going to be not set or empty?

Reply:
I have removed it as is unnecessary.